src/lib/get-messages.js

```diff
@@ -0,0 +1,10 @@
+import defaultLocales from '../../public/locales.json';
```

Comment:
These are just stubs for now. I have them more fleshed out in the other patch but it made sense to separate them.